### ISSUE TITLE
--watch breaks on relative paths

### ISSUE THREAD

**Issue by zackschuster** [sass/dart-sass#787]
if the input/output paths start with a relative specifier, sass crashes:

`sass ./css/style.scss ./css/style.css --watch`

to fix, one must remove the relative specifiers:

`sass css/style.scss css/style.css --watch`

expected behavior to me would be to allow relative path declarations.

**Comment by nex3** [comment]
I can't reproduce this, so I'll need more information in order to help you with your problem. Which Sass implementation are you using, and version of Sass? What's the Sass source that's not behaving how you expect, what output are you expecting, and what output or error are you getting instead? Please make sure your example is as tightly-focused on the problem as possible, so that it's easy for us to understand what's going on.

**Comment by panphora** [comment]
I ran into the same problem today. 

I ran `sass --watch ./project-files/assets/main.sass ./project-files/assets/main.css` on the command line. It would compile after the first run, but then not update on changes to main.sass.

After removing the leading `./` from the input and output arguments, it started watching and compiling on every change.

- I'm using `dart-sass 1.22.9 compiled with dart2js 2.4.0`.
- I'm using a Macbook with macOS Mojave 10.14.5

Here's the full source code needed to replicate this bug for me: 

```
body
  background-color: #eee
```

When I change `background-color: #eee` to `background-color: #ccc`, the `main.css` file doesn't update. However, if I stop the watch command and rerun it, it compiles once (but just once).
